examples/dgmulti_1d/elixir_advection_gauss_sbp.jl

```diff
@@ -0,0 +1,69 @@
+
+using OrdinaryDiffEq
+using Trixi
+
+###############################################################################
+#  setup the equations (for now just linear advection)
+
+advection_velocity = 1.0
+equations = LinearScalarAdvectionEquation1D(advection_velocity)
+
+###############################################################################
+#  setup the GSBP DG discretization that uses the Gauss operators from Chan et al.
+
+surface_flux = FluxLaxFriedrichs()
+dg = DGMulti(polydeg = 3,
+             element_type = Line(),
+             #approximation_type = GaussSBP(), # I want to use GaussSBP here,
```

Comment:
Since the `GaussSBP` is the default here we can probably remove my comment of what I want to use

Reply:
Oops - I had forgotten to make it default in this elixir (I was working in another one). I'll commit that and update the tests.

Reply:
Done now.